static_report/src/sdlc/global.d.ts

```diff
@@ -15,6 +15,7 @@ declare global {
     PIPERIDER_SINGLE_REPORT_DATA: SingleReportSchema;
     PIPERIDER_COMPARISON_REPORT_DATA: ComparisonReportSchema;
     PIPERIDER_METADATA: PipeRiderMetadata;
+    PIPERIDER_INDEX_REPORT_DATA: any;
```

Comment:
It pains me to see this ;) 
Can we perhaps add some definition/shape here?

Reply:
this is removed